### PR TITLE
Newsite

### PR DESCRIPTION
**Pull request by airan1**
I tried to fix some bugs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;500;600;700&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>Document</title>
```
Comment (sdmnc) [inline index.html:10]: You may change title here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="logo">
+            <h3>KMarketer</h3>
+        </div>
+            <ul class="nav_bar">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Product</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a><img alt="svg" src="img/Vector.png"></a></li>
+                <li><a><img alt="svg" src="img/icnshopping-cart.icn-xs.png"></a></li>
```
Comment (sdmnc) [inline index.html:25]: Does it work with such long name? src="img/icnshopping-cart.icn-xs.png"> Maybe to rename?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+             </ul>
+        </nav>
+    </header>
+
+         <div class="main_content">
+            <p class="maintext">We Take Care of<br>      the Future</p>
+            <p class="maintext2">We know how large objects will act, but things on a <br> 
+             small scale just do not act that way.</p>
+        <div class="btn">
+             <a href="#" class="buttonquote">   Get Quote Now   </a>
```
Comment (sdmnc) [inline index.html:35]: Please delete spaces <a href="#" class="buttonquote">   Get Quote Now   </a> and in line 36

**Review by sdmnc — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+       <nav class="nav">
+        <div class="logo">
+            <h3>KMarketer</h3>
+        </div>
+            <ul class="nav_bar">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Product</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a><img alt="svg" src="img/Vector.png"></a></li>
```
Comment (IvanCeban) [inline index.html:24]: your alt doesn't say anything to the user. Use a more meaningful alt

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                <li><a><img alt="svg" src="img/icnshopping-cart.icn-xs.png"></a></li>
+             </ul>
+        </nav>
+    </header>
+
+         <div class="main_content">
+            <p class="maintext">We Take Care of<br>      the Future</p>
```
Comment (IvanCeban) [inline index.html:31]: avoid using `br`. Set the max width instead

File: index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            <p class="maintext2">We know how large objects will act, but things on a <br> 
+             small scale just do not act that way.</p>
+        <div class="btn">
+             <a href="#" class="buttonquote">   Get Quote Now   </a>
+             <a href="#" class="buttonquote2">  Learn More   </a>
```
Comment (IvanCeban) [inline index.html:36]: Find a more meaningfull class names for the buttons and main text. 
`button2` and `maintext2` is a bad practice

File: style.css [hunk 7]
```diff
@@ -0,0 +1,136 @@
+
+.basic-section {
+    width: 100%;
+    height: 100vh;
+    background-image: url("img/cover.jpg");
+    background-position: center;
+    background-size: cover;
+    opacity: 0.9;
```
Comment (IvanCeban) [inline style.css:8]: why did you set the opacity to the whole section?
it applies the opacity to all elements, which is not like on design

File: style.css [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(48, 49, 70); 
+    /*margin: 70px;*/
+    margin-top: 0;
+    color: white;
+    max-width: 1440px;
```
Comment (IvanCeban) [inline style.css:17]: you set max width to the body. 
And set background image to the section. 
Imagine if the screen is wider than 1440px. How will look the background?

File: style.css [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+    margin: 30px;
+    margin-top: 0;
+    font-size: 24px;
+}
+
+
+.main_content{
+    text-align: center;
+    margin-left: 30px;
+    margin-top: -280px;
```
Comment (IvanCeban) [inline style.css:59]: what for did you use negative margin?

File: style.css [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+    font-size: 20px;
+    font-weight: 500;
+}
+.buttonquote, .buttonquote2{
+    color: antiquewhite;
+    font-size: 14px;
+    text-decoration: none;
+    font-weight: 700;
+    border: 2px solid;
+    padding: 10px 25px 10px 25px;
```
Comment (IvanCeban) [inline style.css:82]: suggestion: 
you can optimize your padding from `10px 25px 10px 25px` to `10px 25px`

File: style.css [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+    border-radius: 37px;
+    /* padding-left: 25px; Отступ слева
+    padding-right: 25px; */
+    margin: -10px;
```
Comment (IvanCeban) [inline style.css:86]: avoid using negative margins. Use them only if there is really no other way. In this case you have other options for sure

File: style.css [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+    transition: 0.2s;
+}
+
+.buttonquote{
+    color: antiquewhite;
```
Comment (IvanCeban) [inline style.css:91]: don't write same styles again. You already wrote them in complex selector for both buttons. 
And I really suggest you to add one more common class "button" to both and add all common styles into common class.

File: style.css [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+    border-color: rgb(255, 255, 255);
+}
+
+.buttonquote:hover{
+    background-color:#e58600;
+    border-color: #e58600;
+    color: black;
+}
+
+svg{
```
Comment (IvanCeban) [inline style.css:132]: I bet this didn't work. As you have png images, not svg

**Review by IvanCeban — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (IvanCeban) [reply index.html:25]: agree, better choose a more clear and short name. like `cart-icn`

File: index.html (hunk 4 above)
Comment (IvanCeban) [reply index.html:24]: and the name of the icon sounds weird. `search-icn` sounds much more clear

**Review by IvanCeban — CHANGES_REQUESTED** [review]

**Pull request merged**